src/ui/PaymentDialog/index.svelte

```diff
@@ -57,30 +66,22 @@
       source,
     })
 
-    getPlans()
+    getPlans(isBusiness)
   }
 
-  $: customer = $customer$
-  $: ({ subscription } = customer)
-  $: isNotCanceled = !subscription?.cancelAtPeriodEnd
-  // TODO: make customer data accesible via context
-  $: ({ sanBalance, isEligibleForTrial, annualDiscount } = $customer$)
-  $: name = PlanName[plan.name] || plan.name
-  $: price = name ? formatPrice(plan) : ''
-
   function findDefaultPlan({ name, interval: billing }) {
     return defaultPlan === name && interval === billing
   }
 
-  function getPlans() {
-    const cached = plans.length ? plans : getCachedSanbasePlans()
+  async function getPlans(isBusiness: boolean) {
+    const products = getCachedProducts() ?? (await queryPlans())
+    const plans = isBusiness ? getBusinessPlans(products) : getIndividualPlans(products)
 
-    if (cached) setPlans(cached)
-    else querySanbasePlans().then(setPlans)
+    setPlans(plans)
```

Comment:
This change makes it impossible to display plans passed through `plans` prop, which is critical for a generic behavior.
Solution: keep the old logic. `getIndividualPlans` will be used by default in the future once a business decision has been made.